{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -334,6 +335,16 @@
 CELERY_TIMEZONE = env('CELERY_TIMEZONE', default=TIME_ZONE)  # Use django's timezone by default
 {%- endif %}
 
+# USER AUTH - PASSWORD RESET
+# -----------------------------------------------------------------------------
+# see user.services.send_password_reset
+PASSWORD_RESET_CONFIRM_URL = env('PASSWORD_RESET_CONFIRM_URL',
+                                 default='reset-password/{token}/{uid}')
+
+# Api's typically run on a different name than the frontend app.
+FRONTEND_DOMAIN = env('FRONTEND_DOMAIN', default='http://localhost:8000')
+FROM_EMAIL = env('FROM_EMAIL', default='default@localhost.com')
```

Comment:
The DEFAULT_FROM_EMAIL can be used here. See `settings/production.py`.

Reply:
Should I move this to `settings/common.py` ?